apps/createsamples/utility.cpp

```diff
@@ -1270,7 +1270,7 @@ void cvCreateTrainingSamples( const char* filename,
                 if( showsamples )
                 {
                     cvShowImage( "Sample", &sample );
-                    if( cvWaitKey( 0 ) == 27 )
+                    if( (cvWaitKey( 0 ) & 0xFF) == 27 )
```

Comment:
In cases like here, I would also like to replace 27 by 'ESC' because it simply is more understandable that we are looking the value of the ESC key. If in some future those backend values would change, the code itself would not need updating. 

What you think?